modules/custom/d_commerce/modules/d_commerce_products_list/src/Plugin/facets/url_processor/DCommerceProductListUrlProcessor.php

```diff
@@ -38,6 +38,10 @@ public function buildUrls(FacetInterface $facet, array $results) {
         $view = Views::getView($attributes->get('view_id'));
         $view->setDisplay($attributes->get('display_id'));
         $view->setArguments($result->getUrl()->getRouteParameters());
+        $url = $result->getUrl();
```

Comment:
As i can see, you can do it at the beginning of the `foreach` and replace all occurrences of the `$result->getUrl()`